src/rest/openapi.yaml

```diff
@@ -107,6 +107,24 @@ paths:
                 type: number
                 description: Current state
                 example: 4
+    put:
+      tags:
+        - node
+      summary: Set current Thread state.
+      description: |-
+        Enable and disable the Thread protocol operation. If network interface
+        hasn't been started yet, it will get started automatically.
+      responses:
+        "200":
+          description: Successful operation.
+      requestBody:
+        description: New Thread state
+        content:
+          application/json:
+            schema:
+              type: string
```

Comment:
The GET method returns an integer, but the PUT here gives a string. What about changing the GET format to string as well (I prefer string against integer to make the REST API more readable)

Reply:
The REST API is currently version `0.3.0`. The zero major version means the API is not finalized and we are not responsible for compatibility (see https://semver.org/#spec-item-4). Though you should update the version to `0.4.0` regardless of the major version.

In a later time if you feel confident on the REST API, feel free to propose updating the version to `1.0.0`!